Entities/Vehicles/Common/VehicleCommon.as

```diff
@@ -413,12 +413,19 @@ bool Vehicle_AddLoadAmmoButton(CBlob@ this, CBlob@ caller)
 	{
 		// put in what is carried
 		CBlob@ carryObject = caller.getCarriedBlob();
-		if (carryObject !is null && !carryObject.isSnapToGrid())  // not spikes or door
+		if (carryObject !is null && (!carryObject.isSnapToGrid() || carryObject.getName() == "seed"))  // not spikes or door, allow seeds
```

Comment:
```suggestion
		if (carryObject !is null && !carryObject.hasTag("temp blob"))
```
Instead of checking the seed name we can use a tag. "temp blob" is a tag for build blobs, but doesn't apply to seeds, so we can use it.